tests/vmi_configuration_test.go

```diff
@@ -530,5 +535,144 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+	Describe("VirtualMachineInstance with CPU pinning", func() {
+		var nodes *kubev1.NodeList
+		BeforeEach(func() {
+			nodes, err = virtClient.CoreV1().Nodes().List(metav1.ListOptions{})
+			tests.PanicOnError(err)
+			if len(nodes.Items) == 1 {
+				Skip("Skip cpu pinning test that requires multiple nodes when only one node is present.")
+			}
+		})
+		Context("with cpu pinning enabled", func() {
+			It("should set the cpumanager label to false when it's not running", func() {
+
+				By("adding a cpumanger=true lable to a node")
+				nodes, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: v1.CPUManager + "=" + "false"})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(nodes.Items).To(HaveLen(1))
+
+				node := &nodes.Items[0]
+				node, err = virtClient.CoreV1().Nodes().Patch(node.Name, types.StrategicMergePatchType, []byte(fmt.Sprintf(`{"metadata": { "labels": {"%s": "true"}}}`, v1.CPUManager)))
+				Expect(err).ToNot(HaveOccurred())
+				// wait one heartbeat
+				time.Sleep(60 * time.Second)
```

Comment:
Is this a leftover? The `Eventually` should also wait long enough to handle the worst waiting-case ...

Reply:
done